docs/configuration.md

```diff
@@ -57,6 +55,10 @@ containing element for app view(s). A menu that doesn't use the frame-page
 directive yields a hamburger button that does nothing when clicked. See
 [frame-page.html][] to understand how
 this is constructed.
++ **inactivityTimeout**: the length in minutes a login session can remain
+inactive before the server expires it. A dialog will show during the last
+minute of a session, prompting user action. If no action is taken, the user
```

Comment:
I got curious about best practices about this, specifically whether 1 minute of warning is long enough. Looks like [Harvard's accessibility guidance on this point](https://accessibility.huit.harvard.edu/give-users-control-over-timeouts) uses 1 minute of warning in its example, seems like a good sign that 1 minute is reasonable.